### ISSUE TITLE
NO FUNCIONA EL Deshacer

### ISSUE THREAD

**Issue by LuisGH7** [excalidraw/excalidraw#7554]
no funciona PERDI MUCHA INFORMACION

**Comment by dwelle** [comment]
Hi, can you elaborate on what happened? The data is stored in the browser. If you clear your cookies, or your browser decides to clear the site data, you will lose work. I'd recommend to periodically save to a file if working on a drawing for longer periods of time.

**Comment by LuisGH7** [comment]
no, me refiero al Deshacer dentro mismo, sin querer elimine textos y imagenes y ya no pude recuperarlo mas
